pkgs/ffigen/lib/ffigen.dart

```diff
@@ -8,12 +8,21 @@
 /// https://pub.dev/packages/ffigen for details.
 library ffigen;
 
+export 'src/code_generator/imports.dart' show ImportedType, LibraryImport;
 export 'src/config_provider.dart'
     show
+        CommentType,
```

Comment:
Note to self: We should go and do an API authoring round of FFIgen to make sure use from Dart is nice.